services/storage/src/simcore_service_storage/handlers_files.py

```diff
@@ -91,6 +91,29 @@ async def get_file_metadata(request: web.Request):
         # TODO: once all legacy services are gone, remove the try except, it will default to 404
         return {"error": "No result found", "data": {}}
 
+    if request.headers.get("User-Agent") == "OpenAPI-Generator/0.1.0/python":
+        # LEGACY compatiblity with API v0.1.0
+        # SEE models used in sdk in:
```

Comment:
it is already added in the test since that is the one covering the steps that reproduce that bug. 
That change is mostly related with the models in the client sdk and there are already links for those there